packages/kbn-monaco/src/console/theme.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { makeHighContrastColor } from '@elastic/eui';
+import { darkMode, euiThemeVars } from '@kbn/ui-theme';
+
+import { themeRuleGroupBuilderFactory } from '../common/theme';
+import { monaco } from '../monaco_imports';
+
+const buildRuleGroup = themeRuleGroupBuilderFactory();
+
+const background = euiThemeVars.euiColorLightestShade;
+const methodTextColor = '#DD0A73';
+const urlTextColor = '#00A69B';
+const stringTextColor = '#009926';
+const commentTextColor = '#4C886B';
+const variableTextColor = '#0079A5';
+const booleanTextColor = '#585CF6';
```

Comment:
I was wondering whether we could use the already defined `euiTheme` colors for consistency, like it is done for the [code editor theme](https://github.com/elastic/kibana/blob/8c854632ff532e8aa193393355f627a1633483e4/packages/shared-ux/code_editor/impl/editor.styles.ts#L69) (which I think is used in Painless lab). For example, the method color could be `euiTheme.euiColorAccentText`. But this decision depends on whether we want to keep the theme as close to the ace editor theme as possible.

Reply:
Indeed, the hard coded values are not great, but we can't currently map them one-to-one without changing the colors. I think there are 2 ways to improve this code: either re-use the already existing json code editor theme, but that changes the look of console. Or alternatively, we try to keep the original look but re-use the built-in colors from EUI as much as possible. 
I added a section to https://github.com/elastic/kibana/issues/176926 with follow up work once the initial language definition is implemented.